src/components/Search/components/SearchQueryItemList.tsx

```diff
@@ -0,0 +1,44 @@
+import { useFetchShops } from 'api/search';
+import PreviousButton from 'components/PreviousButton/PreviousButton';
+import { useParams } from 'react-router-dom';
+import styles from 'pages/Search/Search.module.scss';
+import SearchNavigationBar from './SearchNavigationBar';
+import SearchQueryItem from './SearchQueryItem';
+import LoadingView from './LoadingView';
+
+interface Props {
+  address: string,
+  dist: number,
+  placeId: string,
+  placeName: string,
+  score: number,
+  shopId: number,
+  x: string,
+  y: string,
+}
+
+function SearchQueryItemList() {
+  const { searchQuery } = useParams();
+  const { isFetching, data } = useFetchShops(searchQuery as string);
+
+  return (
+    <div>
+      <div className={styles.search}>
+        <SearchNavigationBar>
+          <PreviousButton />
+          <h1 className={styles['search-nav__text']}>{searchQuery}</h1>
+        </SearchNavigationBar>
+        {
+          <div className={styles['search-detail']}>
+            {isFetching ? <LoadingView />
+              // eslint-disable-next-line max-len
+              : data?.data.content.map((shop: Props) => <SearchQueryItem key={shop.shopId} data={shop} />)}
```

Comment:
lint를 반드시 지워야 할 문장 길이는 아닐거라고 생각해요.
```suggestion
          {isFetching
            ? <LoadingView />
            : data?.data.content.map((shop: Props) => (
              <SearchQueryItem key={shop.shopId} data={shop} />
            ))}
```

Reply:
이렇게 해보려고 했는데 자꾸 빨간줄을 긋길래 우선 lint 지웠었는데
 확인해보니 `<div>` 바깥에 불필요한 중괄호가 있었네요 ㅋㅋㅋ😅

수정하겠습니다~